src/components/addBookForm.jsx

```diff
@@ -35,7 +36,13 @@ const AddBook = () => {
       <form onSubmit={addBookHandler}>
         <input type="text" placeholder="Title" value={state.title} name="title" onChange={handleChange} />
         <input type="text" placeholder="Author" value={state.author} name="author" onChange={handleChange} />
-        <button type="submit">Add Book</button>
+        <select type="text" name="category" value={state.category} placeholder="Category" onChange={handleChange} id="input-category">
+          <option value="" disabled>Categories</option>
+          <option value="fiction">Fiction</option>
+          <option value="non-fiction">Non-Fiction</option>
+          <option value="biography">Biography</option>
+        </select>
+        <button type="submit" className="add-butn">Add Book</button>
```

Comment:
- [ ] Your design should match exactly with `Zeplin's` design. Your design doesn't match the original like, your font style, spacing, input fields border, placeholder, font-weight, button colour, button text and border issue. Kindly look at the original design carefully and implement all those changes.

### Your design
![Screenshot from 2023-09-09 23-46-43](https://github.com/yusufyusie/Bookstore-CMS/assets/66476329/ef45dcf9-ddb6-492f-86e1-0e5914ac2c84)